Classes/Lightwerk/SurfCaptain/GitApi/Driver/BitBucketDriver.php

```diff
@@ -41,7 +41,7 @@ public function getRepositories() {
 		$repositories = array();
 		foreach ($this->settings['repositories'] as $command) {
 			$this->apiRequest->setApiUrl($this->settings['apiUrl']);
-			$response = $this->apiRequest->call($command);
+			$response = $this->apiRequest->call($command . '?pagelen=100');
```

Comment:
wenns mehr als 100 sind? Auf jeden Fall würde die Zahl in einer Konstanten verpacken